onnxruntime/core/graph/contrib_ops/bert_defs.cc

```diff
@@ -645,7 +645,7 @@ ONNX_MS_OPERATOR_SET_SCHEMA(
                "past state for key with shape (batch_size, num_heads, past_sequence_length, head_size) for self attention"
                "When past_present_share_buffer is set, "
                "its shape is (batch_size, num_heads, max_sequence_length, head_size). "
-               "The keys buffer is re-ordered in such a way that its virtual sub-tensor of shape "
+               "For CUDA EP, the keys buffer is re-ordered in such a way that its virtual sub-tensor of shape "
```

Comment:
Let's not add EP information in operator spec. If we need to distinguish whether it is re-ordered or not, we probably shall  support shape 4D (batch_size, num_heads, max_sequence_length, head_size) and  5D (batch_size, num_heads, head_size / x, max_sequence_length, x). Or add an attribute to indicate whether past_key is re-ordered or not.

Reply:
Comment used.